docs/mkdocs-base.yml

```diff
@@ -8,7 +8,6 @@ repo_name: wez/wezterm
 edit_uri: edit/main/docs/
 docs_dir: docs
 site_dir: gh_pages
-strict: true
```

Comment:
FWIW, I want to keep strict enabled as catching errors without it is very difficult